benchmarks/benchmarks/stats.py

```diff
@@ -353,3 +353,19 @@ def time_skew(self, order, size, bias):
 
     def time_kurtosis(self, order, size, bias):
         stats.kurtosis(self.x, bias=bias)
+
+
+class BenchQMCDiscrepancy(Benchmark):
+    param_names = ['iterative', 'method']
+    params = [
+        [False, True],
+        ["CD", "WD", "MD", "L2-star",]
+    ]
+
+    def setup(self, iterative, method):
+        np.random.seed(1234)
+        sample = np.random.random_sample((100, 4))
```

Comment:
I would suggest to increase these a bit to really have the effect of the dimension and ns.
```suggestion
        sample = np.random.random_sample((1000, 10))
```